reference/glossary.rst

```diff
@@ -494,19 +494,29 @@ Glossary
         *Work in Progress*
 
     FQDN
-        *Work in Progress*
+        A Fully Qualified Domain Name (FQDN) represents a complete name that specifies the exact location of a host within the :term:`DNS` hierarchy.
 
-    Freedesktop
-        *Work in Progress*
+    Related topic(s): Networking, Web services, Security, Active directory integration, and :term:`OpenLDAP`.
 
     FreeIPA
-        *Work in Progress*
+        Free Identity, Policy, and Audit (FreeIPA) is an open-source security solution for Linux/Unix-like systems that stores 
+        user identities in an :term:`LDAP` directory, manages a :term:`CA`, and enables authentication, policy enforcement, and 
+        auditing through integrations with :term:`SSSD` and :term:`Kerberos`.
+    
+    Related topic(s): Security, and :term:`OpenLDAP`.
```

Comment:
Same issue with this line - needs a bit more indentation to bring it in line with the rest of the definition.